ballerina/client.bal

```diff
@@ -52,7 +52,9 @@ public type Client client object {
     # + return - Metadata of the query execution as an `sql:ExecutionResult[]` or an `sql:Error`
     remote isolated function batchExecute(ParameterizedQuery[] sqlQueries) returns ExecutionResult[]|Error;
 
-    # Executes an SQL query, which calls a stored procedure. This may or may not return results.
+    # Executes an SQL query, which calls a stored procedure. This may or may not
+    # return results. Once the results are processed, invoke the `close` method on the `sql:ProcedureCallResult` or
+    # close the current result by calling `getNextQueryResult()` on the `sql:ProcedureCallResult`.
```

Comment:
```suggestion
    # return results. Once the results are processed, invoke the `close` method on the `sql:ProcedureCallResult` to close the connection.
```
`getNextQueryResult()` only closing the current result set and fetches the next. Does not close the connection.

Reply:
Add a similar warning here, https://lib.ballerina.io/ballerina/sql/1.6.1/classes/ProcedureCallResult#close